pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -101,8 +101,9 @@ type DashboardSpec struct {
 }
 
 type ClusterStatus struct {
-	State   ClusterState `json:"state,omitempty"`
-	Message string       `json:"message,omitempty"`
+	State    ClusterState `json:"state,omitempty"`
+	Message  string       `json:"message,omitempty"`
+	CephName string       `json:"cephName,omitempty"`
```

Comment:
Could we also add a timestamp for last time the status was updated? I don't think it needs to be defined in the `additionalPrinterColumns`, but would be good to have it in the details.